amansachdeva13/kpifire
# Pr with sql tag only

*Include label if *S*Q*L changes were made or an API needs to be run*

### 1. URL to corresponding KPI Fire Task:
https://dev.kpifire.com/#/tasks/52294 (task added on 28 dec)

### 2. Is it a bug or new feature/logic? If bug, why was it not working?
link to QA Master file:


### 3. Steps for Success Test:

1. open project summary tree view
2. check box add to include/ not include sub departments data
**default is to include sub departments**

![Screenshot 2024-08-09 at 5 23 17 AM](https://github.com/user-attachments/assets/0493be36-6d47-42ad-b847-1afc95b616f9)

### 4. Regression Testing: Which areas will/could be affected?:
UI change: y/n
API/Data change: y/n

### 5. for PM: How should this item be listed in release notes?
Is there an in app help tip that requires update based on this change? y/n (add link)
